### PR TITLE
Fix typos in Arrow docs

### PR DESCRIPTION
**Pull request by karthikiyengar**

### CODE REVIEW

**Comment by evilsoft** [comment]
If you want you could run:
`yarn contributors:add karthikiyengar doc` That will add the Documentation icon to your name on all-contributors.

File: docs/src/pages/docs/crocks/Arrow.md [hunk 1]
```diff
@@ -349,7 +349,7 @@ Arrow a b ~> ((c -> a), (b -> d)) -> Arrow c d
 
 `promap` can be used to adapt BOTH ends of an `Arrow` allowing for existing
 `Arrow`s to be reused in places in a flow where the types do not line up. It
-combines both `map` and `conramap` into one operation. Just pass the `contramap`
+combines both `map` and `contramap` into one operation. Just pass the `contramap`
```
Comment (evilsoft) [inline docs/src/pages/docs/crocks/Arrow.md]: The doucmentation [here](https://github.com/evilsoft/crocks/blob/master/docs/src/pages/docs/crocks/Arrow.md#promap) should also be updated to reflect the change.

**Comment by evilsoft** [comment]
Also while you are in there, does it make sense to add links to both `map` and `contramap` on these? using 
```
[`map`](#map) and [`contramap`](#contramap)
```
 as the links?

**Comment by karthikiyengar** [comment]
@evilsoft - I'm a bit confused as to whether I should be making these changes in the `Readme.md` or directly in the documentation file. In case we need it on the Readmes, I'd be happy to move the changes over.

**Comment by evilsoft** [comment]
Just noticed something else. Maybe the wording should be changed from:
> Just pass the [`contramap`](#contramap) function as the first argument and the [`map`](#map) function as the second

To the following:
Just pass the function for [`contramap`](#contramap) as the first argument and the function for [`map`](#map) as the second.

**Comment by evilsoft** [comment]
> I'm a bit confused as to whether I should be making these changes in the Readme.md or directly in the documentation file. In case we need it on the Readmes, I'd be happy to move the changes over.

Both places please. One will be used for the mono repo (the README you just updated) and the docs will be for the published docs. It is manual right now as I do not want to add any automation until we understand how the mono repo is going to be implemented.

File: src/Arrow/README.md [hunk 2]
```diff
@@ -7,11 +7,13 @@ Arrow a b
...
 lazy execution of the function. `Arrow` can be considered a `Strong Profunctor`
 if the underlying data running throw the `Arrow` is a `Pair`, typically in the
-form of `Arrow (Pair a c) (Pair b d)`. This will allow you to split execution
-into two distinct paths, applying `Arrow` to a specific path. The parameters of
-`Arrow` represent the function that it wraps, with the input being on the left,
-and the output on the right. When an `Arrow` wraps an endomorphism, the signature
-typically represents both the input and output.
+form of `Arrow (Pair a c) (Pair b d)`. 
+
+This will allow you to split execution into two distinct paths, applying `Arrow`
```
Comment (karthikiyengar) [inline src/Arrow/README.md]: Just added a newline here (same as docs/Arrow.md)

**Comment by evilsoft** [comment]
Looks great!! Have a squirrel.
image

**Pull request merged**